metricshub-doc/src/site/markdown/getting-started-windows.md

```diff
@@ -0,0 +1,128 @@
+keywords: quick start, getting started
+description: Short step-by-step instructions to follow for installing and configuring MetricsHub in a Windows environment. 
+
+# Quick Start - Windows
+
+<!-- MACRO{toc|fromDepth=1|toDepth=1|id=toc} -->
+
+This quick start guide provides step-by-step instructions for operating MetricsHub and Prometheus in a Windows environment, ensuring you can efficiently monitor your systems.
+
+After completing this quick start, you will have:
+* MetricsHub and Prometheus installed on your machine
+* The MetricsHub Agent configured to collect hardware metrics from your local host and push data to Prometheus
+* MetricsHub and Prometheus up and running
+* Hardware metrics available in Prometheus. 
+
+## Step 1: Install MetricsHub
+
+1. Download the latest package, `metricshub-windows-${project.version}.zip`, from the [MetricsHub Releases](https://github.com/sentrysoftware/metricshub/releases/) page.
+2. Unzip the content of `metricshub-windows-${project.version}.zip` under `C:\Program Files`. There is no need to create a specific subdirectory for `MetricsHub` as the zip archive already contains a `MetricsHub` directory.
+
+> Note: You will need administrative privileges to unzip into C:\Program Files.
+
+## Step 2: Install Prometheus
+
+1. Download [Prometheus](https://prometheus.io/download/).
+2. Extract the content of the archive under `C:\Program Files`. There is no need to create a specific subdirectory for `Prometheus` as the zip archive already contains a `Prometheus` directory.

```

Comment:
It is better to give directives:
```md
* Download [prometheus-<version>.windows-<architecture>.zip](https://prometheus.io/download/).
* Right-click on the downloaded ZIP file and select "Extract All...", enter `C:\Program Files\` when prompted then click Extract. This will extract the directory `prometheus-<version>.windows-<architecture>` to `C:\Program Files\`
* Under `C:\Program Files\` rename the `prometheus-<version>.windows-<architecture>`  directory to `Prometheus`.
```